mergify_engine/context.py

```diff
@@ -529,6 +566,7 @@ def _users_permission_cache_key_for_repo(
             f"{cls.USERS_PERMISSION_CACHE_KEY_DELIMITER}{repo_id}"
         )
 
+    # NOTE(Syffe): this is deprecated
```

Comment:
why?

Reply:
Thought I would note it in order to delete it in a separated PR later ! How should i behave in this kind of case ? just note it for myself, create a linear ticket or just delete it into this PR ?